HealthMonitoring.AcceptanceTests/HealthMonitoring.SelfHost.exe.config

```diff
@@ -13,6 +13,10 @@
     <add key="Monitor.HealthCheckInterval" value="00:00:02" />
     <add key="Throttling.http.json" value="5" />
     <add key="Throttling.http" value="5" />
+    <add key="AdminMonitorId" value="431251bd-b1b7-4a22-834e-c8051210227d" />
+    <add key="AdminMonitorPrivateToken" value="ge49we046hu29m50swn0v3cgg3bwl1i10yj17xp8mnshvfow51alxcui0sany25sg5qqzfplqeeljvusbvxnnj6hvfa8hlarl0z0uohjky6xrwz1zgfh6vc4l6raeq0x" />
+    <add key="PullMonitorId" value="bc4befed-a3da-4da4-9208-586a2d783fa2" />
+    <add key="PullMonitorPrivateToken" value="tjhsudehe53vzsvcxkcpvckba6rd69kyg0uj8ipuzc75y7w7bhdhcviefs4x9l4y896w5f0z78kbdgvnop3gdjhupcyhebi83nmg63ad9dyzp636xokeo2ckhsh99noh" />
```

Comment:
Would it be possible to move credentials out of appconfig into the separate block like that?
It would bring a clear separation between credentials and other settings

```
<configuration>
  <configSections>
    <section name="accessConfiguration" type="System.Configuration.NameValueSectionHandler" />
  </configSections>
  <accessConfiguration>
    <add key="AdminId" value="" />
    <add key="AdminToken" value="" />
    <add key="MonitorId" value="" />
    <add key="MonitorToken" value="" />
  </accessConfiguration>
```